build.gradle

```
@@ -1,6 +1,6 @@
 plugins {
-    id 'fabric-loom' version '0.9.45'
-    id 'io.github.juuxel.loom-quiltflower' version '1.3.0'
+    id 'fabric-loom' version '0.10-SNAPSHOT'
```

```suggestion
    id 'fabric-loom' version '0.10.+'
```

Consider `-SNAPSHOT` deprecated with the new Gradle features.